package.json

```diff
@@ -73,6 +90,6 @@
     "rimraf": "^3.0.2",
     "ts-jest": "29.1.0",
     "typescript": "^4.9.3",
-    "@nx/eslint": "17.1.1"
+    "vscode-material-icons": "^0.1.0"
```

Comment:
This is a legacy dependency. I recommend this [guide](https://github.com/code-pushup/cli/wiki/Updating-Code-PushUp) which updates all packages correctly.